shell/browser/browser.h

```diff
@@ -120,6 +134,13 @@ class Browser : public WindowListObserver {
     base::string16 path;
     std::vector<base::string16> args;
 
+#if defined(OS_WIN)
+    bool enabled = true;
```

Comment:
And this?

Reply:
this enabled indicates whether the launch item entry in the registry has a matching registry key in the startup approved key path